modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-external-network.adoc
 
 :_content-type: PROCEDURE
-[id="virt-attaching-vm-secondary-network-cli_{context}"]
-= Attaching a virtual machine to a secondary network in the CLI
+[id="virt-attaching-vm-additional-network-cli_{context}"]
+= Attaching a virtual machine to an additional network in the CLI
 
-Attach a virtual machine to a secondary network by adding a bridge interface and specifying a network attachment definition in the virtual machine configuration.
+Attach a virtual machine to an additional external network by adding a bridge interface and specifying a network attachment definition in the virtual machine configuration.
```

Comment:
I like the change to "additional".

I would not use the word "external". What I would understand by it is that it exposes the VM to the network outside the cluster, which may or may not be the case, depending on the configuration of the cluster network itself, which is outside the scope of this article.

Reply:
LGTM. Remove "external".

Reply:
Removed.